include/wx/msw/webview_ie.h

```diff
@@ -143,6 +149,9 @@ class WXDLLIMPEXP_WEBVIEW wxWebViewIE : public wxWebView
     void onActiveXEvent(wxActiveXEvent& evt);
     void onEraseBg(wxEraseEvent&) {}
 
+    //Establish EmulationLevel for RunScript IE
+    static bool MSWSetEmulationLevel(bool defaultLevel = false);
```

Comment:
It should be renamed now that it doesn't set a level any more. `MSWSetDefaultEmulationLevel()` would work, but it would be logical to use `bool defaultLevel = true` for such function. If you want to keep `false` as default value, you need to call it `MSWSetNonDefaultEmulationLevel()` or `MSWSetModernEmulationLevel()` or something like that.